### ISSUE TITLE
Missing A in docs

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/hvplot#849]
https://pyviz-dev.github.io/hvplot/

I believe it should be "A familiar and high-level ..."

![image](https://user-images.githubusercontent.com/42288570/186656423-b4e50c57-01ff-42b7-8a0b-85e79934179f.png)

**Comment by jbednar** [comment]
Agreed! A PR would be very helpful, preferably combined with other docs fixes.
